lib/Controller/Report.php

```diff
@@ -1090,6 +1106,68 @@ public function savedReportExport(Request $request, Response $response, $id, $na
         ));
     }
 
+    /**
+     * Saved Report Convert Form
+     * @param Request $request
+     * @param Response $response
+     * @param $id
+     * @return \Psr\Http\Message\ResponseInterface|Response
+     * @throws AccessDeniedException
+     * @throws GeneralException
+     * @throws NotFoundException
+     * @throws \Xibo\Support\Exception\ControllerNotImplemented
+     */
+    public function convertSavedReportForm(Request $request, Response $response, $id)
+    {
+        $savedReport = $this->savedReportFactory->getById($id);
+
+        if (!$this->getUser()->checkDeleteable($savedReport)) {
```

Comment:
Do we want delete permissions for this operation? or edit permissions?

Reply:
This has gone? We aren't checking permissions at all on convert now. Is that right?